public/index.html

```diff
@@ -5,5 +5,6 @@
 </head>
 <body>
   <h1 class="hello-world">Hello, World!</h1>
+  <h1 class="hello-from-alex">Hello, from Alex!</h1>
```

Comment:
You can place comments directly on the line of code. For instance say you saw a typo in the class I used here, you could add a comment here letting me know.